autoPyTorch/pipeline/components/setup/augmentation/image/base_image_augmenter.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict, Optional
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+
+from imgaug.augmenters.meta import Augmenter
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.setup.base_setup import autoPyTorchSetupComponent
+
+
+class BaseImageAugmenter(autoPyTorchSetupComponent):
+    def __init__(self) -> None:
+        super().__init__()
+        self.augmenter: Optional[Augmenter] = None
+
+    def get_image_augmenter(self) -> Augmenter:
+        """
+        Get fitted augmenter. Can only be called if fit()
+        has been called on the object.
+        Returns:
+            BaseEstimator: Fitted augmentor
+        """
+        if self.augmenter is None:
+            raise AttributeError("Can't return augmenter for {}, as it has not been"
+                                 " fitted yet".format(self.__class__.__name__))
+        return self.augmenter
+    #
+    # def check_requirements(self, X: Dict[str, Any], y: Any = None) -> None:
```

Comment:
Why is this commented out :D?

Reply:
I'm sorry I had copied first in case I need requirements. As augmenter has no requirements, it is not needed but I forgot to delete